src/backend/gl/src/lib.rs

```diff
@@ -577,6 +577,19 @@ impl Device {
                     gl.PatchParameteri(gl::PATCH_VERTICES, num as gl::types::GLint);
                 }
             },
+            Command::CopyBuffer(src, dst, src_offset, dst_offset, size) => {
+                let gl = &self.share.context;
+                unsafe {
+                    // TODO: check capabilities (during encoding ?)
```

Comment:
yes, if possible. It seems rather... difficult to proceed with this way in GL if we can't copy buffers